File: src/dataset/Java/isogram_exercism-java-uzilan_robiworks/claude/ClaudeSonnet4_isogram_v1.java
class IsogramChecker {

    boolean isIsogram(String phrase) {
        int len = phrase.length();
        boolean[] seen = new boolean[26];
        
        for (int i = 0; i < len; i++) {
            char c = phrase.charAt(i);
            if (c == ' ' || c == '-') {
                continue;
            }
            
            // Convert to lowercase index
            int index;
            if (c >= 'A' && c <= 'Z') {
                index = c - 'A';
            } else if (c >= 'a' && c <= 'z') {
                index = c - 'a';
            } else {
                continue;
            }
            
            if (seen[index]) {
                return false;
            }
            seen[index] = true;
        }
        return true;
    }
}

File: src/dataset/c/isogram_Exercism_HeitorMP/claude/ClaudeSonnet4_isogram_v1.c
#include "isogram.h"
#include <stdint.h>

bool is_isogram(const char phrase[])
{
    if (phrase == NULL)
        return false;
    
    uint32_t seen = 0;
    
    for (const char *p = phrase; *p; ++p)
    {
        unsigned char c = *p;
        if ((c >= 'A' && c <= 'Z') || (c >= 'a' && c <= 'z'))
        {
            c |= 0x20; // Convert to lowercase
            int bit_pos = c - 'a';
            uint32_t mask = 1U << bit_pos;
            if (seen & mask)
                return false;
            seen |= mask;
        }
    }
    return true;
}